app/assets/stylesheets/application.css.scss

```diff
@@ -135,3 +163,46 @@ input#front-search-button {
   text-align: center;
   color: white !important;
 }
+
+.tag {
+  color: blue;
+}
+
+.venues {
+  width: 60%;
+  background: {
+    color: #170563;
```

Comment:
Color literals like `#170563` should only be used in variable declarations; they should be referred to via variable everywhere else.